tools/c7n_org/c7n_org/cli.py

```diff
@@ -486,14 +487,14 @@ def run_account_script(account, region, output_dir, debug, script_args):
 @click.option('-a', '--accounts', multiple=True, default=None)
 @click.option('-t', '--tags', multiple=True, default=None, help="Account tag filter")
 @click.option('-r', '--region', default=None, multiple=True)
-@click.option('--echo', default=False, is_flag=True)
-@click.option('--serial', default=False, is_flag=True)
+@click.option('-e', '--echo', default=False, is_flag=True)
+@click.option('-d', '--debug', default=False, is_flag=True)
```

Comment:
can we just add `-d/--debug` to the existing `--serial` otherwise this is a breaking change